src/constants.ts

```diff
@@ -31,7 +31,8 @@ export enum GaugeVSCodeCommands {
     RepeatExecution = 'gauge.execute.repeat',
     SwitchProject = 'gauge.specexplorer.switchProject',
     QuickPickOnExecution = 'gauge.quickPick.onExecution.open',
-    ExtractConcept = 'gauge.extract.concept'
+    ExtractConcept = 'gauge.extract.concept',
+    CreateAndSendTextToInstallTerminal = "gauge.createAndSendText.terminal"
```

Comment:
`CreateAndSendTextToInstallTerminal ` sounds like a very loaded name. Can we have something simpler, ex `ExecuteInTerminal`?

Also consider simplifying `gauge.createAndSendText.terminal`